### PR TITLE
WhiteSource Local config file

### PR DESCRIPTION
**Pull request by mukul-seagate11**
## Problem Statement:
WhiteSource Local config file will be created to scan & detect vulnerabilities based on local component configurations/Artifacts/code extensions etc.

### CODE REVIEW

File: whitesource-components.config [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+checkPolicies=false
+forceCheckAllDependencies=false
+forceUpdate=false
+forceUpdate.failBuildOnPolicyViolation=false
+updateInventory=true		  
+
+###########################################################################################
+# Includes/Excludes Glob patterns - Please use only one exclude line and one include line #
+###########################################################################################
+includes=**/*.c **/*.cc **/*.cp **/*.cpp **/*.cxx **/*.c++ **/*.h **/*.hpp **/*.hxx
```
Comment (cdeshmukh) [inline whitesource-components.config:29]: py not included ?

Comment (mukul-seagate11) [reply whitesource-components.config]: Done, added in whitesource config file

Comment (jjxsg) [reply whitesource-components.config]: Should we include shell scripts as well?

Comment (mukul-seagate11) [reply whitesource-components.config]: No, not required

**Review by cdeshmukh — APPROVED** [review]

**Review by sandeepsgt — APPROVED** [review]

**Pull request merged**